qemu/tests/set_link.py

```diff
@@ -203,7 +202,7 @@ def set_link_test(linkid, link_up, change_queues, operstate_always_up=False):
                 expect_status = win_media_connected
         guest_interface_operstate_check(expect_status, guest_ifname,
                                         change_queues)
-        if params.get("os_type") == "windows":
+        if params.get("os_type") == "windows" and link_up:
```

Comment:
`if params.get("os_type") == "windows" and  expect_status == win_media_connected`